### ISSUE TITLE
[Question]: Expect changes to vstsagentpackage.azureedge.net URL's?

### ISSUE THREAD

**Issue by dribblor** [microsoft/azure-pipelines-agent#5075]
### Describe your question

Are your azureedge.net domains being retired (similar to https://github.com/dotnet/core/issues/9671) and will this affect the agent package URL ([ref](https://learn.microsoft.com/en-us/azure/devops/pipelines/agents/windows-agent?view=azure-devops#im-running-a-firewall-and-my-code-is-in-azure-repos-what-urls-does-the-agent-need-to-communicate-with))?

### Versions

n/a

### Environment type (Please select at least one enviroment where you face this issue)

- [x] Self-Hosted
- [ ] Microsoft Hosted
- [ ] VMSS Pool
- [ ] Container

### Azure DevOps Server type

dev.azure.com (formerly visualstudio.com)

**Comment by dribblor** [comment]
Hello?

**Comment by agarshubh** [comment]
Yes the old URL has been retired, please follow the blog for more information.
https://devblogs.microsoft.com/devops/cdn-domain-url-change-for-agents-in-pipelines/
